ex02.exs

```diff
@@ -1,6 +1,27 @@
 
 defmodule Ex02 do
-
+  def start_link(), do: Agent.start_link( fn -> 0 end )
+  
+  def get_and_update(counter) do
+    Agent.get_and_update(counter, fn count -> { count, count + 1 } end)
+  end
+
+  def new_counter(count) do
+    { :ok, counter_pid } = Agent.start_link( fn -> count end )
+    counter_pid
+  end
+
+  def next_value(counter) do
+    get_and_update(counter)
+  end
+
+  def new_global_counter() do
```

Comment:
## -2

I'm grading you a little stricter because you're an 8k student.

Two things here.

First, I'd like to see the process name extracted out into an attribute:

~~~ elixir
@me Counter
~~~

Second: is there a reason that line 23 doesn't call `get_and_update` defined on line 5? (The one on line 15 does)